.github/workflows/mirror-images.yaml

```diff
@@ -18,7 +18,7 @@ jobs:
           REPO: ghcr.io/${{ github.repository_owner }}/shipwright-samples
         run: |
           for IMAGE in \
-            library/golang:1.16 \
+            library/golang:1.17 \
```

Comment:
I suggest to only add 1.17 and retain 1.16 here. The image sync that we perform here gets consumed in https://github.com/shipwright-io/sample-go/blob/main/docker-build/Dockerfile and we do not necessarily need to update this now.